ajax/patterns-01-en.json

```
@@ -381,7 +381,7 @@
     "MANDATORY": "",
     "TANDP": "0"
   }, {
-    "NAME": "<a href=\"https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-information-architecture-specification/usage-canadaca-design.html\">Who has to use this design system</a>",
+    "NAME": "<a href=\"https://design.canada.ca/architecture/usage-canadaca-design.html\">Who has to use this design system</a>",

```

```suggestion
    "NAME": "<a href=\"{{ site.url }}/architecture/usage-canadaca-design.html\">Who has to use this design system</a>",

```
Using a variable in the URL allow us to test if the table works locally, in the staging area and in production, without us having to change the urls manually.

---

I have updated the urls.